### DISCUSSION TITLE
[Question] Isn't Winauth the first free, open source, two factor authenticator for Windows?

### DISCUSSION THREAD

**Discussion by spidirs** [2fast-team/2fast#104] [General]
Hi there,

Just double-checking whether the statement that "**2fast** is the first free, open source, two factor authenticator for Windows [...] with the ability to store the sensitive data encrypted at a place of your choice" is indeed correct.

I ask because I've been using [Winauth](https://github.com/winauth/winauth) for a good while; it's been around since 2010, apparently, and it allows users to export password-protected (zip but also PGP) text file, if I'm not mistaken.

It's quite possible that I'm missing something, though, so I'm merely flagging this as a potential issue.

Thanks,

G.

**Comment by jp-weber** [comment]
Hey,

you're right, I only had the active larger providers in mind at the time of the release and not winauth, which has already been discontinued. I will adapt the description, however.

**Reply by spidirs** [reply]
Hi,

No worries, and thanks for replying.
